easybuild/easyconfigs/q/QScintilla/QScintilla-2.14.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,80 @@
+# Author: Pavel Grochal (INUITS)
+# Update v2.14.1: Pavel Tománek (INUITS)
+# License: GPLv2
+
+easyblock = 'ConfigureMake'
+
+name = 'QScintilla'
+version = '2.14.1'
+
+homepage = 'https://www.riverbankcomputing.com/software/qscintilla'
+description = "QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control"
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://www.riverbankcomputing.com/static/Downloads/QScintilla/%(version)s/']
+sources = ['QScintilla_src-%(version)s.tar.gz']
+checksums = ['dfe13c6acc9d85dfcba76ccc8061e71a223957a6c02f3c343b30a9d43a4cdd4d']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('PyQt-builder', '1.15.4'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('PyQt5', '5.15.10'),
+]
+
+# run qmake instead of configure
+skipsteps = ['configure']
+
+start_dir = 'src'
+
+prebuildopts = 'export CPATH=$EBROOTQT5/include/QtWidgets:$EBROOTQT5/include/QtPrintSupport:$CPATH && '
+# do not install to Qt dirs but to QScintilla dirs
+prebuildopts += (
+    "sed -i "
+    r"-e 's|\$\$\[QT_INSTALL_HEADERS\]|%(installdir)s/include|g' "
+    r"-e 's|\$\$\[QT_INSTALL_LIBS\]|%(installdir)s/lib|g' "
+    "%(start_dir)s/features/qscintilla2.prf && "
+)
+prebuildopts += (
+    "sed -i "
+    r"-e 's|\$\$\[QT_HOST_DATA\]|%(installdir)s/data|g' "
+    r"-e 's|\$\$\[QT_INSTALL_DATA\]|%(installdir)s/data|g' "
+    r"-e 's|\$\$\[QT_INSTALL_HEADERS\]|%(installdir)s/include|g' "
+    r"-e 's|\$\$\[QT_INSTALL_LIBS\]|%(installdir)s/lib|g' "
+    r"-e 's|\$\$\[QT_INSTALL_TRANSLATIONS\]|%(installdir)s/trans|g' "
+    "%(start_dir)s/qscintilla.pro && "
```

@pavelToman To fix CI you probably need to use `%(namelower)s` here:
```suggestion
    "%(start_dir)s/%(namelower)s.pro && "
```